content/partials/specs/versions-macos-silicon-xcode-16.md

```diff
@@ -0,0 +1,137 @@
+---
+description: A list of tools available out-of-the-box on Codemagic macOS Apple silicon build machines using Xcode version 16.0.
+title: macOS Apple silicon build machine specification (Xcode 16.0+)
+aliases:
+
+weight: 17
+---
+
+## Hardware
+
+- VM on Mac mini M1 `3.2GHz Quad Core / 8GB`
+- VM on Mac mini M2 `3.5GHz Quad Core / 8GB`
+
+## System
+
+- System version `macOS 14.5 (23F79)`
+- Kernel version `Darwin 23.5.0`
+- Disk `294GB (Free Space: 84GB)`
+
+## Pre-installed tools
+
+- Android tools `/usr/local/share/android-sdk`
+- Android NDK `25.1.8937393`
+- appium `2.5.0`
+- aws `2.15.38`
+- carthage `0.39.1`
+- cocoapods `1.15.2`
+- cordova `12.0.0`
+- curl `8.6.0`
+- docker `N/A`
+- ew-cli `0.11.0`
+- fastlane `2.220.0`
+- firebase `11.21.0`
+- flutter `3.22.2 ($HOME/programs/flutter)`
+- gem `3.5.9`
+- gh `2.48.0`
+- git `2.44.0`
+- Google Cloud SDK `404.0.0`
+- gradle `8.1.1`
+- gsutil `5.14`
+- homebrew `4.2.18`
+- ionic `7.2.0`
+- jq `1.7.1`
+- ktlint `1.2.1`
+- node `20.12.2`
+- npm `10.5.0`
+- python `3.8.13`
+- python3 `3.8.13`
+- ruby `3.0.4p208`
+- ssh `9.6p1`
+- sudo `1.9.13p2`
+- swiftgen `6.6.3`
+- tar `3.5.3`
+- ucd `0.11.10`
+- unzip `6.00`
+- wget `1.24.5`
+- yarn `1.22.22`
+- yq `4.43.1`
+- zip `3.0`
+
+## Android emulators
+
+Android emulators are unavailable on Apple silicon machines due to the Apple Virtualization Framework not supporting nested virtualization. Please use a Linux instance.
+
+## Java versions
+
+- **22.0.1** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-22.jdk/Contents/Home`
+- **21.0.3** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-21.jdk/Contents/Home`
+- **17.0.11** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home`
+- **11.0.23** (default) JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home`
```

Comment:
should be 17

Reply:
I mean default version is 17 starting this Monday

Reply:
good catch! the problem is that we still have 11 as default in the base image and only in builder we switch it to 17 but our vm-tools scripts is not aware of this. I will fix this.

Reply:
actually, the other specs also have 11 as default which I understand is incorrect. I will update all then.